src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParser.cs

```diff
@@ -6243,7 +6306,11 @@ private void LoadSSPILibrary()
                     {
                         // use local for ref param to defer setting s_maxSSPILength until we know the call succeeded.
                         UInt32 maxLength = 0;
+#if MANAGED_SNI
+                        if (0 != SNIProxy.Singleton.InitializeSspiPackage(ref maxLength))
+#else
                         if (0 != SNINativeMethodWrapper.SNISecInitPackage(ref maxLength))
+#endif // MANAGED_SNI
```

Comment:
In all of these #if...else...endif cases, it'd be nice if we could follow the same scheme we do elsewhere in the repo.  You have two files that expose the same surface area, you make calls to that surface area, and then you just include whichever file is appropriate for a given build.  That way, the calling code remains clean, it's obvious from the .csproj which support is being used, etc.